style.css

```diff
@@ -85,12 +106,166 @@ header nav .toolbar {
   line-height: 39px;
 }
 
+.skill img {
+  margin-right: 8px;
+}
+
 .headline ul li img {
   height: 24px;
 }
 
-.feeler {
-  position: absolute;
-  right: 0;
-  bottom: 0;
+/* -----------------work-section------------- */
+
+.section-container {
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+hr {
+  width: 80%;
+}
+
+.work-section-container h2 {
+  font-family: 'crete Round', sans-serif;
+  text-align: center;
+  border: 1px solid black;
+  font-weight: 400;
+  font-size: 40px;
+  color: #1c1a19;
+  width: 300px;
+  margin: 24px auto;
+}
+
+.work-container {
+  padding-right: 12px;
+  padding-left: 12px;
+  padding-bottom: 10px;
+  margin: auto;
+  row-gap: 50px;
+  display: grid;
+  grid-template-columns: auto;
+  justify-content: center;
+}
+
+.item .write-up {
+  padding-bottom: 30px;
+}
+
+.item {
+  padding: 150px 0 0;
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("image/background_img/Img-Placeholder.png");
+  background-position: center;
+}
+
+.work-item-1 {
+  display: flex;
+  flex-direction: column;
+  background-image: none;
+}
+
+.project-btn {
+  display: block;
+  padding: 12px 16px;
+  width: 100%;
+  color: #fff;
+  background-color: #ff6b00;
+  text-align: center;
+  font-size: 18px;
+}
+
+.work-item-1 .project-btn {
+  display: inline;
+}
+
+.item h3 {
+  max-width: 295px;
+}
+
+.write-up p {
+  color: #000;
+  font-weight: 500;
+}
+
+.write-up a,
+.item a {
+  text-decoration: none;
+}
+
+.item ul {
+  display: flex;
+  list-style: none;
+  padding-left: 0;
+}
+
+.item ul li {
+  background-color: gray;
+  border: 1px solid rgb(202, 190, 190);
+  padding: 8px;
+  margin-right: 8px;
+}
+
+.res-btn a {
+  color: #fff;
+  text-decoration: none;
+}
```

Comment:
[OPTIONAL]
- Please add `border: none;` to remove that border around your button so that it matches that of the Figma.
### Your implementation
![image](https://user-images.githubusercontent.com/45460192/164891866-39d999f5-9fd8-4af9-a59e-33c714fb7467.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/164891878-a688f266-85e2-44e8-8984-7635e5fdd5d4.png)

Reply:
ok... thanks